### ISSUE TITLE
Rule disable comments do not take the language into account

### ISSUE THREAD

**Issue by ChrisMBarr** [microsoft/vscode-eslint#2029]
I am using a set of Angular HTML template rules, and when I want to disable one of them it inserts a disable comment using a JS/TS syntax instead of HTML

![Image](https://github.com/user-attachments/assets/8224aa30-7e05-4e39-9514-3e1bc4029e1b)

**Actual Output**
```html
// eslint-disable-next-line @angular-eslint/template/label-has-associated-control
```

**Expected Output**
```html
<!-- eslint-disable-next-line @angular-eslint/template/label-has-associated-control -->
```

Originally was reported https://github.com/angular-eslint/angular-eslint/issues/2547 but I was directed to this project instead.

**Comment by dbaeumer** [comment]
Dups https://github.com/microsoft/vscode-eslint/issues/1074
